kalite/main/tests/api_tests.py

```diff
@@ -1,10 +1,50 @@
 """
 """
+import json
+from django.core.urlresolvers import reverse
+
 from ..models import VideoLog, ExerciseLog
 from kalite.facility.models import Facility, FacilityUser
-from kalite.testing.base import KALiteTestCase
+from kalite.testing.base import KALiteTestCase, KALiteClientTestCase
 from kalite.testing.client import KALiteClient
 
+
+class ContentItemApiViewTestCase(KALiteClientTestCase):
```

Comment:
I think for theses tests, you would be better off writing a topic node to the database first, and then retrieving it, then you can be sure about what you find.